src/save_index.cpp

```diff
@@ -46,6 +46,16 @@ void replace_space2underbar(std::string &name) {
 	std::replace(name.begin(),name.end(),' ','_');
 }
 
+/** Replace the spaces in the filter */
+std::string replace_filter(const std::string* filter)
+{
+    std::string filter_value = *filter;
```

Comment:
Ohh, sorry, I misread the line. Okay, in that case, I thnk it's actually redundant to have both this and the following line - you're copying the string twice in a row when you really only need to copy it once.